README.md

```diff
@@ -1,37 +1,34 @@
-## Welcome to GitHub Pages
+## JuliaGeo
+JuliaGeo is an organization that contains a number of related Julia projects for manipulating, querying, and processing geospatial geometry data. We aim to provide a common interface between geospatial packages.
 
-You can use the [editor on GitHub](https://github.com/JuliaGeo/juliageo.github.io/edit/master/README.md) to maintain and preview the content for your website in Markdown files.
+### Get involved
+Communcation is mostly done on the Julia [Discourse](https://discourse.julialang.org/c/domain/geo). More short-lived interaction is found on Julia [Slack #geo](https://slackinvite.julialang.org/). Feel free to create issues, PRs on packages.
 
-Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
+We're currently discussing changes to our common GeoInterface.
 
-### Markdown
+### Packages
+Here's a partial listing of some of the existing libraries in julia that you can use for working with geospatial data. Not all of them are listed under the JuliaGeo organization, but some of them might get moved over.
 
-Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for
+#### Wrappers for external libraries
+Using the excellent [BinaryBuilder.jl](https://github.com/JuliaPackaging/BinaryBuilder.jl) we try to provide prebuilt binaries for most external libraries.
+- [LibGEOS.jl](https://github.com/JuliaGeo/LibGEOS.jl) is a wrapper to GEOS for performing geospatial operations on vector geometries
+- [Proj4.jl](https://github.com/JuliaGeo/Proj4.jl) is a wrapper to PROJ.4 for performing cartographic projections
+- [GDAL.jl](https://github.com/JuliaGeo/GDAL.jl) is a wrapper to GDAL (Geospatial Data Abstraction Library) for reading and writing raster and vector datasets; [ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl) builds on top of GDAL.jl for a more Julian user experience. [GeoArrays.jl](https://github.com/evetion/GeoArrays.jl) builds on top of both to provide the easiest, but simplest experience.
+- [NetCDF.jl](https://github.com/JuliaGeo/NetCDF.jl) and [NCDatasets.jl](https://github.com/Alexander-Barth/NCDatasets.jl) provides a high-level and a medium-level interface for writing and reading netcdf files.
+- [GMT.jl](https://github.com/joa-quim/GMT.jl) provides a wrapper for working with the [G]eneric [M]apping [T]ools (GMT) library in julia.
+- [LibSpatialIndex.jl](https://github.com/JuliaGeo/LibSpatialIndex.jl) provides functionality for spatially indexing kD bounding box data (based on [libspatialindex](https://github.com/libspatialindex/libspatialindex))
```

Comment:
Native counterpart https://github.com/alyst/SpatialIndexing.jl would also be good to mention.